src/entries/popup/pages/swap/index.tsx

```diff
@@ -454,8 +456,12 @@ export function Swap({ bridge = false }: { bridge?: boolean }) {
       if (!assetToBuy) tokenToBuyInputRef.current?.openDropdown();
       setAssetToSellInputValue('');
       setAssetToBuyInputValue('');
+      if (asset) {
+        setHasRequestedMaxValueAssetToSell(true);
```

Comment:
instead of a flag that triggers the `useEffect` can't we just put `setAssetToSellMaxValue();` here?

Reply:
ah I see 🫠 hate it but seems like only refactoring everything to do that